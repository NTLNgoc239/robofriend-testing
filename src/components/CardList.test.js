import { shallow, mount, render } from "enzyme";
import React from "react";
import CardList from './CardList';
import toJson from "enzyme-to-json";

it("Expect to return Card component", () => {
  const mockRobot = [
    {
      id: 1,
      name: "John Snow",
      username: 'JohnJohn',
      email: "john@example.com"
    }
  ]
  const wrapper = shallow(<CardList robots={mockRobot} />);
  expect(
    toJson(wrapper)
  ).toMatchSnapshot();
});