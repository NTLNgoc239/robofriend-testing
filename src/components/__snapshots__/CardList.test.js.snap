// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to return Card component 1`] = `
<div>
  <Card
    email="john@example.com"
    id={1}
    key="0"
    name="John Snow"
  />
</div>
`;
